src/push_api_clientpy/platformclient.py

```diff
@@ -103,11 +107,26 @@ class BatchUpdateDocuments:
     addOrUpdate: list[Document]
     delete: list[BatchDelete]
 
+@dataclass
+class BackoffOptions:
+    retry_after: int = DEFAULT_RETRY_AFTER
+    max_retries: int = DEFAULT_MAX_RETRIES
+    time_multiple: int = DEFAULT_TIME_MULTIPLE
+
 
 class PlatformClient:
-    def __init__(self, apikey: str, organizationid: str):
+    def __init__(self, apikey: str, organizationid: str, backoff_options: BackoffOptions, session = requests.Session()):
```

Comment:
Unless I'm reading this wrong, this looks like it's a breaking change?

Should it be this instead (just like `Source`)?
```suggestion
    def __init__(self, apikey: str, organizationid: str, backoff_options: BackoffOptions = BackoffOptions(), session = requests.Session()):
```

Reply:
Oh yes, good catch! I applied this suggestion ✅